tests/glsl-intrinsic/atomic/atomicCounter.slang

```diff
@@ -48,87 +43,79 @@ bool testAtomicUint()
 {
     // ensure the code emits for `one` index into [3] for 12/4
     // CHECK_GLSL-DAG-N: {{.*}}_data_0[3]{{.*}}
-    // ensure the code emits for `two` index into [4] for 16/4
-    // CHECK_GLSL-DAG-N: {{.*}}_data_0[4]{{.*}}
     return true
 
-// CHECK_GLSL-DAG-N: atomicCounterIncrement(one0._data_0[3]
-// CHECK_SPV-DAG-N: OpAtomicIAdd
+// CHECK_GLSL-N: atomicCounterIncrement(one_0._data_0[3]
+// CHECK_SPV-N: OpAtomicIAdd
```

Comment:
One of my concerns about this is that OpAtomicIAdd may find a matching line generated from the prvious functions.
I think you can isolate the searching range by checking the function name first, something like this:
```
// CHECK_SPV-LABEL: counterAsParam
// CHECK_SPV: OpAtomicIAdd

// CHECK_SPV-LABEL: testAtomicUint
// CHECK_SPV: OpAtomicIAdd
```
You can use LABEL and drop the "-N" for OpAtomicIAdd.

Reply:
I will change the syntax, your concern is 100% correct, the tests sometimes pass when they should fail. I will remedy this (using label and changing the tests), I think having good tests for atomic_uint will be quite helpful since its a big compiler magic trick